Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hyper-proxy"
-version = "0.9.0"
+version = "0.9.1"
```

Comment:
I recommend to up this to `0.10.0` because this `rustls` has a higher MSRV.

Reply:
I didn't spot that - thanks! I'll update the version number.